openstudiocore/src/openstudio_lib/OSDocument.cpp

```diff
@@ -604,6 +598,11 @@ namespace openstudio {
 
   void OSDocument::createTab(int verticalId)
   {
+    if (m_runningApplyMeasureNow) {
+      m_runningApplyMeasureNow = false;
```

Comment:
This seems to get reset when accepting changes during apply measure but not if you hit cancel on the dialog